src/main/java/org/mifos/pheevouchermanagementsystem/util/VoucherManagementEnum.java

```diff
@@ -0,0 +1,16 @@
+package org.mifos.pheevouchermanagementsystem.util;
+
+public enum VoucherManagementEnum {
+    SUCCESS_RESPONSE_CODE("00"),
+    FAILED_RESPONSE_CODE("01"),
+    SUCCESS_RESPONSE_MESSAGE("Request successfully received by Pay-BB"),
```

Comment:
you can create this enum with both code and message. you have to modify the constructor accordingly.

Eg:  ``` SUCCESS("00", "Request successfully received by Pay-BB")

constructor will be

VoucherManagementEnum(String code, String message) {
        this.code = code;
        this.message = message;
    }```

Reply:
Implemented this